tests/kernel/msgq/msgq_api/src/test_msgq_contexts.c

```diff
@@ -316,6 +357,56 @@ void test_msgq_alloc(void)
 	zassert_true(ret == -EINVAL, "Invalid request");
 }
 
+/**
+ * @brief Get message from an empty queue
```

Comment:
/**
* @brief Test get message from an empty queue
* @details 
* - A thread get message from an empty queue will get a -ENOMSG if timeout is set to K_NO_WAIT
* - A thread get message from an empty queue will be blocked if timeout is set to K_FOREVER